Sources/MapboxNavigation/OverviewCameraOptions.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+/**
+ Options, which are used to control what `CameraOptions` parameters will be modified by
+ `NavigationViewportDataSource` in `NavigationCameraState.overview` state.
+ */
+public struct OverviewCameraOptions {
```

Comment:
It's basically part of `FollowingCameraOptions`. Maybe it's better to extract these parameters to a common structure (something like `CameraUpdateOptions`)? It would allow generalizing decisions on camera changes in `NavigationViewportDataSource`

Reply:
Even though those properties have similar naming each of them will be used in two different states, so I don't think that common place will give us significant benefits. Also there is a chance that this part will be changed based on end-users feedback. I'll keep in mind your proposal though.